ramenpos/Podfile

```diff
@@ -6,7 +6,9 @@ target 'RamenPos' do
     use_frameworks!
     
     # Pods for RamenPos
-    pod 'SPIClient-iOS', '~> 2.4.0'
+    #pod 'SPIClient-iOS', '~> 2.4.4'
+    pod 'SPIClient-iOS', :git => 'https://github.com/AssemblyPayments/spi-client-ios', :branch => 'feature/v2.4.x'
```

Comment:
Is this supposed to be checked in? Just checking :)

Reply:
After releasing I will change it